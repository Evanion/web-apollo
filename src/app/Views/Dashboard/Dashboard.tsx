import * as React from 'react';
import {Link} from 'react-router-dom';

const DashboardPage = () => (
  <div>
    <h1>Dashboard page</h1>
    <Link to={'/login'}>Login</Link>
    <Link to={'/reset-password'}>Reset password</Link>
  </div>
);

export default DashboardPage;
