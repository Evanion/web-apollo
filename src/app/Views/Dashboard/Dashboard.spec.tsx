import * as React from 'react';
import { shallow } from 'enzyme';
import Dashboard from './Dashboard';

describe('Dashboard', ()=>{
  it('should render correctly', () => {
    const render = shallow(<Dashboard />);
    expect(render).toMatchSnapshot();
  });
})
