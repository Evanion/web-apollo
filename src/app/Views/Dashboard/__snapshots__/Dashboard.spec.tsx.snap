// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard should render correctly 1`] = `
<div>
  Dashboard page
</div>
`;
